arithematic-quiz-2/src/components/QuizCreateForm/index.js

```diff
@@ -0,0 +1,120 @@
+import { useState } from "react";
+
+import Form from "react-bootstrap/Form";
+
+const defaultQuizData = {
+  minLimit: "",
+  maxLimit: "",
+  timeLimit: "",
+  questionCount: "",
+  operations: [],
+};
+const OPERATORS = ["+", "-", "*", "/"];
+
+const QuizCreateForm = ({ addQuiz }) => {
+  const [quizData, setQuizData] = useState(defaultQuizData);
+
+  const handleChange = (e) => {
+    const { name, value, checked } = e.target;
+    if (name === "operations") {
+      if (checked) {
+        setQuizData({
+          ...quizData,
+          operations: [...quizData.operations, value],
+        });
+      } else {
+        setQuizData({
+          ...quizData,
+          operations: quizData.operations.filter((item) => item !== value),
+        });
+      }
+    } else {
+      setQuizData({ ...quizData, [name]: parseInt(value) });
+    }
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    addQuiz({ ...quizData, id: Date.now() });
+    setQuizData({ ...defaultQuizData });
+  };
+
+  return (
+    <div className="col-md-6">
+      <div className="row input-form">
+        <div className="text-center h4">Quiz Generation</div>
+        <br />
+        <Form onSubmit={handleSubmit}>
+          <div className="row">
+            <Form.Group className="col-md-6  mb-3" controlId="minLimit">
```

Comment:
these all form control are repeated, can we make a constant and iterate over it?